SnakeController/SnakeController.cpp

```diff
@@ -1,5 +1,14 @@
+
 #include "SnakeController.hpp"
 
+#define constW 'W'
+#define constF 'F'
```

Comment:
preprocesor polecalybm nazywac CONST_W CONST_F itd wedlug https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#enum1-prefer-enumerations-over-macros 

to zawsze moze na przyszlosc zostac :) ale fajny pomysl wgl z podmiana tego !